codespell_lib/data/dictionary.txt

```diff
@@ -17487,6 +17534,7 @@ identfier->identifier
 identfiers->identifiers
 identiable->identifiable
 idential->identical
+identic->identical
```

Comment:
Comment only: `identic` seem like a valid word. Anyway, it seem to be the same as `identical`
Mostly invalid in en_GB but seem valid in some en_US dictionary.
IMO, should be fine to use a more acceptable word rather than `identic`.
Anyway, most dictionaries including chrome browser considered it as misspelled.
* https://www.merriam-webster.com/dictionary/identic
* https://www.dictionary.com/browse/identic

Reply:
@luzpaz Maybe we should move `identic->identical` to rare dictionary.